internal/stacks/stackruntime/internal/stackeval/stubs/unknown.go

```diff
@@ -364,6 +364,34 @@ func (u *unknownProvider) DeleteState(providers.DeleteStateRequest) providers.De
 	}
 }
 
+// PlanAction implements providers.Interface.
+func (u *unknownProvider) PlanAction(request providers.PlanActionRequest) providers.PlanActionResponse {
+	return providers.PlanActionResponse{
```

Comment:
you could leave a TODO here to revisit once we know what we want to do with deferrals in regards to actions?